electron/constants.ts

```diff
@@ -24,6 +24,13 @@ export const URLS_PUBLIC_WITNET_NODES = [
   '3.21.74.162:21338',
 ]
 
+export const OLD_WITNET_NODE_IPS: Array<string> = [
```

Comment:
I had to check why we still need this IPs. Adding "legacy configuration" to the name may make it easier to understand
```suggestion
export const LEGACY_CONFIGURATION_WITNET_NODE_IPS: Array<string> = [
```